crates/node/src/network_server/grpc_svc_handler.rs

```diff
@@ -27,34 +28,31 @@ use restate_types::nodes_config::Role;
 use restate_types::protobuf::node::Message;
 use restate_types::storage::StorageCodec;
 
-pub struct NodeSvcHandler<T> {
+pub struct NodeSvcHandler {
     task_center: task_center::Handle,
     cluster_name: String,
     roles: EnumSet<Role>,
     health: Health,
-    connections: ConnectionManager<T>,
 }
 
-impl<T: TransportConnect> NodeSvcHandler<T> {
+impl NodeSvcHandler {
     pub fn new(
         task_center: task_center::Handle,
         cluster_name: String,
         roles: EnumSet<Role>,
         health: Health,
-        connections: ConnectionManager<T>,
     ) -> Self {
         Self {
             task_center,
             cluster_name,
             roles,
             health,
-            connections,
         }
     }
 }
 
 #[async_trait::async_trait]
-impl<T: TransportConnect> NodeSvc for NodeSvcHandler<T> {
+impl NodeSvc for NodeSvcHandler {
```

Comment:
Should we bring the name NodeCtl back for this service? 😄

Reply:
Can do. Then we would rename this service into `NodeCtl` and the `AliveNodeSvc` into `CoreNodeSvc`?